apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts

```diff
@@ -1014,42 +1012,28 @@ export const useNativeAssetImage = ({
   const mimeType = needsGif ? 'image/gif' : 'image/webp'
   const headers = useMemo(
     () => ({
-      Accept: responseType === 'binary' ? mimeType : 'text/plain',
       'X-Encoded-Mimetype': mimeType,
```

Comment:
in that case, should it replace the `Accept` to use directly the `mimeType` expected, and drop the custom header?

Reply:
IIii1iI don't know, not how @pedromtcosta did it. We can change it I guess? Right now it should be failing though, since he already deployed it like this.